src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -260,5 +277,41 @@ private static void UpdatePropertyForLocaleManifests(string propertyName, string
                 manifest.GetType().GetProperty(propertyName).SetValue(manifest, value);
             }
         }
+
+        /// <summary>
+        /// Compares the hashes of the original and updated manifests and returns true if the new manifest has an updated SHA256 hash.
+        /// </summary>
+        /// <param name="oldManifests">The original manifest object model.</param>
+        /// <param name="newManifests">The updated manifest object model.</param>
+        /// <returns>A boolean value indicating whether the updated manifest has new changes compared to the original manifest.</returns>
+        private bool CheckForUpdatedInstallerHash(Manifests oldManifests, Manifests newManifests)

```

Comment:
I'd just pass the lists of installers directly since that's all this function needs

Reply:
I agree, changed to passing in installerManifest directly